src/app/external-cluster-wizard/steps/external-cluster/provider/gke/template.html

```diff
@@ -72,35 +107,55 @@
         Kubernetes Version is <strong>required</strong>.
       </mat-error>
     </mat-form-field>
+  </div>
+
+  <div [fxFlex]="isDialogView() ? '100%' : '50%'"
+       fxLayout="column"
+       fxLayoutGap="10px">
+    <ng-container *ngIf="!isDialogView()">
+      <mat-card-header class="km-no-padding">
+        <mat-card-title>Initial Node Pool</mat-card-title>
+      </mat-card-header>
+
+      <mat-form-field>
+        <mat-label>Node Pool Name</mat-label>
+        <input matInput
+               type="text"
+               [formControlName]="Controls.InitialNodePoolName" />
+        <mat-hint>Initial Node Pool Name can't be changed</mat-hint>
+      </mat-form-field>
+    </ng-container>
 
-    <km-autocomplete *ngIf="isDialogView()"
-                     label="Machine Type"
-                     [options]="machineTypes"
-                     [formControlName]="Controls.MachineTypes">
-      <mg-container hint>
-        Select vCPU cores and memory
-      </mg-container>
-    </km-autocomplete>
-
-    <km-autocomplete *ngIf="isDialogView()"
-                     label="Disk Type"
-                     [options]="diskTypes"
-                     [formControlName]="Controls.DiskTypes">
-      <mg-container hint>
-        SSD persistent disk is better for random IOPS or for streaming throughput with low latency.
-      </mg-container>
-    </km-autocomplete>
-
-    <km-number-stepper *ngIf="isDialogView()"
-                       hint="Boot disk size is permanent."
+    <km-combobox filterBy="name"
+                 inputLabel="Select Machine Type..."
+                 hint="Select vCPU cores and memory"
+                 [label]="machineTypeLabel"
+                 [options]="machineTypes"
+                 [formControlName]="Controls.MachineTypes">
+      <div *option="let machineType">
+        {{machineType.name+" ("+machineType.vcpus +" vCPUs "+machineType.memory+"MB RAM)"}}
```

Comment:
**Changes should be tested on both (Wizard/MD dialog)**

Don't add computation on template instead create a method in `ts` file which can be also passed as [valueFormatter] in component. 


Show both name + computation on Selection Too

![machine-type (display-name)](https://user-images.githubusercontent.com/17727069/187875725-be650314-90c7-4ce3-8d84-af861d878e7d.png)